pkgs/top-level/python-packages.nix

```diff
@@ -299,7 +303,7 @@ in {
   cdecimal = callPackage ../development/python-modules/cdecimal { };
 
   cfn-flip = callPackage ../development/python-modules/cfn-flip { };
- 
+
```

Comment:
please don't change random lines

Reply:
sorry my editor did this automatically when I saved the file (whitespace).